README.md

```diff
@@ -396,14 +396,71 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored in the `TokenManager`.
+
+By default:
+
+  - the user will be signed out using an API method 
+  - the page location will not be changed. 
+  
+When calling `signOut` with the default options, it is recommended to reload the page after the promise resolves to ensure all user data is cleared from memory and the app is properly loaded in an unauthenticated state.
+
+Since the API method relies upon an XHR call and the ability to set 3rd-party cookies, it may fail in certain scenarios, leaving the user's remote session open. For this reason, we generally recommend using the redirect method over the API method. The redirect method sets the window location to Okta where the session will be fully cleared.
+
+`signOut` takes the following options:
+
+* `redirect` - If `true`, enables the logout redirect method. By default, this will redirect the user to an Okta-hosted login page. Logging back in from this page may send the user to their Okta user homepage rather than the application.
+* `redirectUri` - Enables logout redirect using a custom URI to which the browser will be redirected after logout. This URI must be whitelisted as a `Logout Redirect Uri` in your application's general settings. It is common to use your application URI here.
```

Comment:
I'm not comfortable with this having the same label as the 'redirectUri' for login.  I'd suggested `postLogoutRedirectUri` - it's wordy, but clear, and matches the "post_logout_redirect_uri" language of OIDC.

Reply:
Yes that is a very fair point

Reply:
I have updated to use `postLogoutRedirectUri`